lib/cmd/common/build.js

```diff
@@ -119,29 +120,61 @@ function argsToPayload(args) {
 }
 
 /**
- * Mutates the arguments object in order to support --branch --commit, turning them into gitRef
+ * Mutates the arguments object in order to support --git-branch --git-commit and --git-tag, turning them into gitRef
  * @param  {Object} args Arguments object
  */
 function mutateGitRefArgs(args) {
-  if (args.branch) {
+  // Workaround for current millicore behavior:
+  // If no git- params are present, build from branch:master
+  var gitParams = [
+    'git-commit',
+    'git-tag',
+    'git-branch'
+  ];
+  if (!_.any(gitParams, _.propertyOf(args))) {
     args.gitRef = {
       type: 'branch',
-      value: args.branch
+      value: 'master'
```

Comment:
Yes, pointing to master is a workaround for now because millicore is building on top of the gitRef used on the last time `fhc build` is invoked when you don't specify anything, so it's something users wouldn't expect.

There's some discussion about this on the JIRA, I'm going to add this also to the top comment.